sunpy/visualization/wcsaxes_compat.py

```diff
@@ -43,7 +43,22 @@ def is_wcsaxes(axes):
 
 def gca_wcs(wcs, fig=None):
     """
-    Get the current axes, and return a WCSAxes if possible
+    Get the current axes, and return a WCSAxes if possible.
```

Comment:
What if you take my branch and make a PR on to it?  Would that be the
cleanest way to fix my branch and ultimately the docs?

On Mon, Jul 20, 2015 at 10:07 AM, Jack Ireland <jackireland@gmail.com>
wrote:

> I am pretty sure all the docs will need another run through after today.
>
> On Mon, Jul 20, 2015 at 10:01 AM, Jack Ireland <jackireland@gmail.com>
> wrote:
>
>> I put a few question marks in the PR.  I thought it best to point out
>> where documentation is missing, rather than have me attempt to explain
>> something I don't know about. Consider the question marks to be place
>> holders!!  So that people who actually know about the question marks can do
>> something about it.
>>
>>
>> On Mon, Jul 20, 2015 at 9:50 AM, Stuart Mumford <notifications@github.com
>> > wrote:
>>
>>> In sunpy/visualization/wcsaxes_compat.py
>>> <https://github.com/sunpy/sunpy/pull/1504#discussion_r34995812>:
>>>
>>> > @@ -43,7 +43,22 @@ def is_wcsaxes(axes):
>>> >
>>> >  def gca_wcs(wcs, fig=None):
>>> >      """
>>> > -    Get the current axes, and return a WCSAxes if possible
>>> > +    Get the current axes, and return a WCSAxes if possible.
>>>
>>> this is my bad :(
>>>
>>> —
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/sunpy/sunpy/pull/1504/files#r34995812>.
>>>
>>
>>
>